docs/fsharp/what-is-fsharp.md

```diff
@@ -0,0 +1,120 @@
+---
+title: What is F#
+description: Learn about what the F# programming language is and what programming with it is like.
+ms.date: 08/03/2018
+---
+# What is F# #
+
+F# is a functional programming language that makes it easy to write correct and maintainable code.
+
+F# programming primarily involves defining types and functions that are type-inferred and generalized automatically. This allows your focus to remain on the problem domain and manipulating its data, rather than the details of programming.
+
+```fsharp
+open System // Get access to functionality in System namespace.
+
+// Takes a name and produces a greeting.
```

Comment:
Only because this is introductory material, I would add to the comment that this defines a function.